packages/search/integration-tests/utils/database.ts

```diff
@@ -0,0 +1,124 @@
+import { TSMigrationGenerator } from "@mikro-orm/migrations"
+import { MikroORM, SqlEntityManager } from "@mikro-orm/postgresql"
+import * as ProductModels from "@models"
```

Comment:
question: Shouldn't this be searchModel?

```suggestion
import * as SearchModels from "@models"
```